appstudio-utils/util-scripts/fetch-test-data.sh

```diff
@@ -21,14 +21,14 @@ for tr in $( pr-get-tr-names $PR_NAME ); do
   if [[ ! -z "${data}" ]]; then
       result_found=1
       task_name=$( tr-get-task-name ${tr} )
-      echo "${data}" | jq > $( json-data-file test ${task_name} )
+      echo "$data" | jq '{ "tests": . }' > json-input-file "$tr-tests"
```

Comment:
I think there's a problem here if there are multiple tasks with results. Firstly, each one would write to the same filename, since $tr is the same. Secondly, even if it didn't do that, I think conftest would merge all the values of the top level "tests" key and we'd end up losing all but the last one.

So I think somehow we need to merge the results to the file each time a new task run is added. I expect jq can handle that, but we might need some @lcarva level jq to do it.

At the end we might have something like this:

```
   {
     "tests": {
          "$task_name_1": { ... },
          "$task_name_2": { ... },
     }
   }
```

Reply:
Should not be more complicated than:
```suggestion
      echo "$data" | jq "{ \"tests\": { \"$task_name\": . } }" > json-input-file "$tr-tests"
```

Reply:
From slack discussion: Let's not move the test result data into the input. Reasoning:
* Avoid more complicated changes incompatible with existing rego
* Avoid spending time on a problem that is likely to go away if we can in future access result data in the pipelinerun attestion